server/src/config.js

```diff
@@ -2,9 +2,23 @@ import "./boot.js";
 import getNodeEnv from "./config/getNodeEnv.js";
 import getDatabaseUrl from "./config/getDatabaseUrl.cjs";
 
-export default {
+const development = {
+  awsAccess: { key: process.env.AWS_ACCESS_KEY },
```

Comment:
We also need to update the `.env.example` file with placeholders for these new keys!

Reply:
.env example file has been updated